src/main/java/frc/robot/Robot.java

```diff
@@ -44,20 +44,17 @@ public void configureButtonBindings() {
         .whenReleased(intake.retract(SpinDirection.COUNTERCLOCKWISE));
 
     // Climber:
-    // Move with variable speed from triggers
-    driveController.x().whileHeld(parallel("Move", leftClimber.move(trigger), rightClimber.move(trigger)));
+    copilotController.x().whileHeld(parallel("Move", leftClimber.move(trigger), rightClimber.move(trigger)));
+    copilotController.y().whileHeld(parallel("Rotate", leftClimber.rotate(trigger), rightClimber.rotate(trigger)));
```

Comment:
We'll have to discuss these with the drive team, but I wonder if using the joysticks would be more intuitive here.